### DISCUSSION TITLE
IIS Reverse Proxy

### DISCUSSION THREAD

**Discussion by Fareroo7** [strapi/strapi#6672] [Legacy Topics]
Hello,

I cannot get strapi to work with a reverse proxy with IIS.

In the admin panel as soon as I try to list all users I get an HTTP error code 400

https://{domain}/content-manager/explorer/plugins::users-permissions.user/count?_limit=10&_sort=username:ASC&_start=0 (400)

But at the server, the request was handled successfully.
[2020-06-14T19:23:30.126Z] debug GET /content-manager/explorer/plugins::users-permissions.user?_limit=10&_sort=username:ASC&_start=0 (31 ms) 200

I think there is a missing configuration for the reverse proxy...

Any ideas?

**Comment by nba9898** [comment]
im having the same problem. did you ever find a solution for this?

**Reply by derrickmehaffy** [reply]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
